lib/sync-DataSetModel.js

```diff
@@ -406,25 +429,32 @@ var self = {
     self.doListHandler(dataset_id, datasetClient.queryParams, function (err, records) {
       if (err) return cb(err);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)